dash/components/src/Keychains/KeychainCard.tsx

```diff
@@ -179,15 +179,12 @@ const KeychainCard: React.FC<Props> = ({
           {props.mode === `assign_to_child` && props.schedule && (
             <div
               className={cx(
-                `flex justify-center items-center gap-2 px-4`,
+                `flex justify-center items-center gap-2 px-4 @container/schedule`,
```

Comment:
very excited to see container queries introduced, and it's a super good application of them for this tricky UI. 👍 💯